configs/sandpack-contents/component-theming/skeleton.js

```diff
@@ -0,0 +1,117 @@
+module.exports = {
+  App: `import {
+    Text,
+    Box,
+    Flex,
+    Skeleton,
+    Button,
+    IconButton,
+    useColorMode,
+  } from "@chakra-ui/react";
+  import { FaMoon, FaSun } from "react-icons/fa";
+  import { useState } from "react";
+  
+  export default function App() {
+    const [loading, setLoading] = useState(true);
```

Comment:
Let's rename these to isLoading, setIsLoading since generally booleans should have `is` prefix.